packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/ConditionalFormatting.tsx

```diff
@@ -244,6 +249,15 @@ const ConditionalFormatting: FC<ConditionalFormattingProps> = ({
         [handleChange, config],
     );
 
+    const hasPercentageFormat = useMemo(() => {
```

Comment:
should we move this to the conditional formatting utils and reuse it in `convertFormattedValue` ?

Reply:
ah, great idea, updated!